connectors/utils.py

```diff
@@ -693,6 +693,8 @@ def __init__(self):
         self.extraction_config = ExtractionService.get_extraction_config()
         if self.extraction_config is not None:
             self.host = self.extraction_config.get("host", None)
+            self.timeout = self.extraction_config.get("timeout", 30)
+            self.chunk_size = self.extraction_config.get("stream_chunk_size", 65535)
```

Comment:
Just curious why `65535`, but not `65536`? :D

Reply:
Fixed, could you re-review? Sorry haha